RELEASE_NOTES.md

```diff
@@ -0,0 +1,51 @@
+# What's new in YugabyteDB Voyager
+#### New features, key enhancements, and bug fixes
+Included here are the release notes for the [YugabyteDB Voyager](https://docs.yugabyte.com/preview/migrate/) v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
+
+## v1.3 - May 29, 2023
+##### Key enhancements
+- Export data for mysql and oracle is now 2-4x faster. You can use the env var `BETA_FAST_DATA_EXPORT=1` to leverage this performance improvement. Most features such as migrating partitioned tables, sequences, etc are supported in this mode. Please refer to the [documentation](https://docs.yugabyte.com/preview/migrate/migrate-steps/#export-data) for more details.
```

Comment:
consider adding this, as it is updated to internal release notes also:
`Note that for Postgres, the default mode (pg_dump) is faster, and hence, this flag is not recommended to be used.`